share/locale/en.json

```diff
@@ -1,165 +1,165 @@
 {
-    "config_onion_service": "Configuring onion service on port {0:d}.",
-    "preparing_files": "Preparing files to share.",
+    "config_onion_service": "Setting up onion service on port {0:d}.",
+    "preparing_files": "Preparing OnionShare.",
```

Comment:
OnionShare now supports "receive mode" instead of just "share mode". Really what this means is, if you're using "share mode", it does stuff like compresses the files into a zip. So I think actually the original is more accurate because there is no "Preparing OnionShare" when using receive mode.

Reply:
I see this is nondescript, "Preparing an OnionShare."